### PR TITLE
Feature/hardware wallet

### PR DESCRIPTION
**Pull request by AbstractFruitFactory**

### CODE REVIEW

File: src/app/modules/account/AccountManager.ts [hunk 1]
```diff
@@ -146,15 +173,32 @@ export default class AccountManager {
...
+        }
+
+        store.commit('setActiveAccount', account)
+
+        // Subscribe to updates
+        transferSubscription = account.identity.account.transferSystem.transactionSubject
+            .subscribe(transactionUpdateSubject)
+    }
+
+    public subscribeToTransferEvents(func: any) {
```
Comment (MuncleUscles) [inline src/app/modules/account/AccountManager.ts]: The type here could be more precise

**Review by MuncleUscles — COMMENTED** [review]
I believe remote identity stuff won't work correctly with hardware wallets, because the code in https://github.com/radixdlt/desktop-wallet/blob/release/2.0.0-beta.2/src/app/server/RadixServer.ts has not been updated to use stuff from `send-transactions.ts`

File: src/app/modules/account/AccountManager.ts (hunk 1 above)
Comment (alexmedkex) [reply src/app/modules/account/AccountManager.ts:200]: that one should be removed as it's not being used

Comment (AbstractFruitFactory) [reply src/app/modules/account/AccountManager.ts]: fixed!

**Comment by AbstractFruitFactory** [comment]
> I believe remote identity stuff won't work correctly with hardware wallets, because the code in https://github.com/radixdlt/desktop-wallet/blob/release/2.0.0-beta.2/src/app/server/RadixServer.ts has not been updated to use stuff from `send-transactions.ts`

You're right, fixed this!

**Pull request merged**